### PR TITLE
[IMP] live chat: update ratings doc

### PR DESCRIPTION
**Pull request by jero-odoo**
This doc is still in progress, I am waiting to hear back from PE/PO about a few details (specifically publishing the ratings on the website/portal).

### CODE REVIEW

**Comment by jero-odoo** [comment]
Additional changes made, still waiting for input from expert before further review.

File: content/applications/websites/livechat/ratings.rst
Comment (larm-odoo) [reply content/applications/websites/livechat/ratings.rst:76]: Also, is "livechat" case sensitive- does it not work if it is typed in as "Livechat"?

File: content/applications/websites/livechat/ratings.rst
Comment (larm-odoo) [inline content/applications/websites/livechat/ratings.rst:87]:
```suggestion
Make any desired changes or additions to this page, then click :guilabel:`Save` in the top right of the webpage
```

You click on the Save, so I swapped the GUI labels.

File: content/applications/websites/livechat/ratings.rst
Comment (larm-odoo) [inline content/applications/websites/livechat/ratings.rst:88]:
```suggestion
editor. The website editor side panel closes, and the webpage remains on the screen.
```

File: content/applications/websites/livechat/ratings.rst
Comment (larm-odoo) [inline content/applications/websites/livechat/ratings.rst:91]:
```suggestion
:menuselection:`Site --> Content --> Pages`. Click the checkbox to the left of `livechat` in the list of pages
```

File: content/applications/websites/livechat/ratings.rst
Comment (larm-odoo) [inline content/applications/websites/livechat/ratings.rst:93]: When I do this, the flow is different. After clicking the checkbox next to livechat, it highlights the line. Then, I click on the box next to Is Published- the first click just 'highlights' that box (it turns white) but it does not check the box. The second time I click, the box is checked off, but then there is no Save option- you only see 'new' where 'save' previously was, because doing the double check makes the page published. If I tried to click once, and it is not checked off and click save nothing happens.

File: content/applications/websites/livechat/ratings.rst
Comment (larm-odoo) [inline content/applications/websites/livechat/ratings.rst:92]:
```suggestion
to select the page and highlight the line. Then, click the checkbox under the column labeled :guilabel:`Is Published`. The field with the checkbox is highlighted in white. Click the checkbox a second time to activate the :guilabel:`Is Published` box. The webpage is now published. 
```

File: content/applications/websites/livechat/ratings.rst
Comment (larm-odoo) [inline content/applications/websites/livechat/ratings.rst:93]:
```suggestion
```

File: content/applications/websites/livechat/ratings.rst
Comment (larm-odoo) [inline content/applications/websites/livechat/ratings.rst:100]: I didn't read through the rest yet, but this should be explained. if it is explained further down, I would do a link within the doc so user's can click on "individual ratings can be manually selected to be hidden from the public" be the text tht links to that section of instructions.

Comment (larm-odoo) [reply content/applications/websites/livechat/ratings.rst:100]: ok zi saw the section below- I would link it still, even though it's not far below.

File: content/applications/websites/livechat/ratings.rst
Comment (larm-odoo) [inline content/applications/websites/livechat/ratings.rst:128]:
```suggestion
To hide a rating, go to :menuselection:`Live Chat app --> Reports --> Customer Ratings`. Click on the
```

This was another note someone gave me that I started to use, which was specify when the item is an app or not, so most things would start with the X app, then the rest. Just a thought- not necessary but it does add clarification (especially since we jump to the website app in this doc, too).

File: content/applications/websites/livechat/ratings.rst
Comment (larm-odoo) [inline content/applications/websites/livechat/ratings.rst:129]:
```suggestion
kanban card for the rating to be hidden. On the individual rating's detail page, check the box labeled
```

File: content/applications/websites/livechat/ratings.rst
Comment (larm-odoo) [inline content/applications/websites/livechat/ratings.rst:139]: Just a general comment, so I added it on this last line! It might be good to add an explanation of how to organize the feedback in kanban columns, sorted by rating. I think that might be helpful than the default view. It might be good for them to know. This might be covered in a reporting doc if that's in the works, but if not, I like to add things like that which could be very helpful, esp. for a first time. They are probably asked to aggregate the #'s, etc, and that might be easy and add value.

**Review by larm-odoo — CHANGES_REQUESTED** [review]
Great doc and nice improvements! There was one small section where what I had to do was a little off, so I changed that part, and I clarified a few spots where I was confused (figuring if I am someone else could be =).  And I added a suggestion of something to add with the reporting part. LMK if you have any questions on my comments.

File: content/applications/websites/livechat/ratings.rst
Comment (jero-odoo) [reply content/applications/websites/livechat/ratings.rst:75]: The field is called "Page Title" but it is actually used to create the Page URL. After the page has been created, the actual page title can be changed. I've updated for clarification. 👍

**Comment by jero-odoo** [comment]
Hi @tiku-odoo, this one is ready for your review. Thanks!

File: content/applications/websites/livechat/ratings.rst
Comment (tiku-odoo) [inline content/applications/websites/livechat/ratings.rst:86]: this seems to be broken in 16.3

File: content/applications/websites/livechat/ratings.rst
Comment (tiku-odoo) [inline content/applications/websites/livechat/ratings.rst:110]: Sam may have you change this to display the default title

File: content/applications/websites/livechat/ratings.rst
Comment (tiku-odoo) [inline content/applications/websites/livechat/ratings.rst:19]: these shouldn't be indented

File: content/applications/websites/livechat/ratings.rst
Comment (tiku-odoo) [inline content/applications/websites/livechat/ratings.rst:71]: missing period

File: content/applications/websites/livechat/ratings.rst
Comment (tiku-odoo) [inline content/applications/websites/livechat/ratings.rst:39]: Also option to publish and to managers for internal use?

File: content/applications/websites/livechat/ratings.rst
Comment (tiku-odoo) [inline content/applications/websites/livechat/ratings.rst:69]: manually added

File: content/applications/websites/livechat/ratings.rst
Comment (tiku-odoo) [inline content/applications/websites/livechat/ratings.rst:124]: What about different views?

**Review by tiku-odoo — COMMENTED** [review]
@jero-odoo This doc is ready for you. I've made a few suggestions, looks excellent as it stands. 

Nice work, and looking forward to seeing this published. 

*I did a functionality review through 16.3 runbot

Tim 
:+1:

**Comment by jero-odoo** [comment]
Thank you! Sending to @odoo/us-doc-review for tech review and final.

**Comment by StraubCreative** [comment]
hi @jero-odoo i need approvals from @tiku-odoo and @larm-odoo first before @samueljlieber or I review, thanks!

**Review by tiku-odoo — APPROVED** [review]
Looks good, just the two open comments outstanding that need clarification. Good to move forward! :+1:

**Review by larm-odoo — APPROVED** [review]
Looks good- I didn't actually click approve last time! Sorry!

**Review by StraubCreative — APPROVED** [review]
Great doc, thanks @jero-odoo 👍 

@robodoo r+

**Pull request closed without merging**
